Server/ReasnAPI/ReasnAPI/Services/ImageService.cs

```diff
@@ -130,18 +150,18 @@ public ImageDto GetImageById(int id)
         return imageDto;
     }
 
-    public IEnumerable<ImageDto> GetImagesByObjectId(int objectId, ObjectType objectType)
+    public IEnumerable<ImageDto> GetImagesByUserId(int userId)
     {
         var images = context.Images
-            .Where(image => image.ObjectId == objectId && image.ObjectType == objectType)
+            .Where(image => image.ObjectId == userId && image.ObjectType == ObjectType.User)
             .ToList();
 
         if (!images.Any())
         {
             throw new NotFoundException("Images not found");
         }
 
-        var imageDtos = images.Select(image => image.ToDto()).AsEnumerable();
+        var imageDtos = images.ToDtoList().AsEnumerable();
 
         return imageDtos;
     }
```

Comment:
Is it needed? We have getting by object id with object type specified

Reply:
Same for getting for event

Reply:
i got request from @mkoper02 to create GetImagesByUserId instead of get by objectid and object type

Reply:
and i removed method get by object id and object type